### DISCUSSION TITLE
Is there a simple manual how to use a file

### DISCUSSION THREAD

**Discussion by xtraa** [dirkwhoffmann/virtualc64#772] [Q&A]
Hi, I searched git and the website but I could not find a manual on how to actually load a .prg file. Not sure where to put it, since there is an app but no folder defined. I figured out how to unlock the window when your mouse pointer is trapped instead of reset my computer, but if I do a load "*",8 (or ,8,1) I don't know where it is searching for the files.

**Comment by dirkwhoffmann** [comment]
- The emulator does not seek media files in specific folders. To add a media file (disk, tape, etc.), simply drag and drop the file into emulator window. 
- The easiest way to release the mouse is to just shake it (assuming the "Release by shaking" option is enabled in the emulator settings). More details about the mouse settings can be found [here](https://dirkwhoffmann.github.io/vAmiga/docs/References/Preferences/Controls.html). It's a page from the vAmiga documentation, but it pretty much applies to VirtualC64, too.
